.github/workflows/dev-image-update.yaml

```diff
@@ -76,28 +76,6 @@ jobs:
     needs:
       - bump_version
     steps:
-      - name: Checkout Develop branch of jade-data-repo
-        uses: actions/checkout@v3
-        with:
-          ref: develop
-          token: ${{ secrets.BROADBOT_TOKEN }}
-      - name: 'Checkout datarepo-helm-definitions repo'
-        uses: actions/checkout@v3
-        with:
-          repository: 'broadinstitute/datarepo-helm-definitions'
-          token: ${{ secrets.BROADBOT_TOKEN }}
-          path: datarepo-helm-definitions
-      - name: "Build new delevop docker image"
-        uses: broadinstitute/datarepo-actions/actions/main@0.74.0
-        with:
-          actions_subcommand: 'gradlebuild'
-          sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
```

Comment:
Why do we need to build two docker images? We are already building an image for the release candidate